deploy/packages/external-libraries.yaml

```diff
@@ -28,7 +28,6 @@ packages:
       - cudnn@7.3.0
       # - freeimage@3.18.0
       - glew
-      - gmsh+metis~mpi+oce+openmp+shared@4.4.1
```

Comment:
This has moved to `environments/externals.yaml`

Reply:
I updated `environment/externals.yaml`. Should I revert the initial changes of this PR @matz-e ?